### ISSUE TITLE
Sonarr media request wont change to processing/available (sonarr v4)

### ISSUE THREAD

**Issue by m-klecka** [Fallenbagel/jellyseerr#614]
### Description

After requesting a show, it gets succesfully pushed to sonarr and downloaded. It is then available in Jellyfin, as anticipated, but never changes to "processing" (when downloading) or "Available" when available.

### Version

1.7.0

### Steps to Reproduce

Request a show, wait for it to download.

### Platform

desktop

### Device

lenovo notebook

### Operating System

windows 11

### Browser

firefox

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
Can you hop onto discord. This is a configuration issue and ill help you fix it. We dont really provide support on issues as its hard to follow through

**Comment by Fallenbagel** [comment]
Solution: turn on season folders. I'm closing this issue
